### ISSUE TITLE
Add additional video player keyboard shortcuts

### ISSUE THREAD

**Issue by feross** [feross/webtorrent-desktop#118]
These are the shortcuts that I personally use a lot in VLC:

- [x] Skip forward 10 seconds (`CMD` `ALT` `right`)
- [x] Skip back 10 seconds (`CMD` `ALT` `left`)
- [x] Increase video speed (`CMD` `+`)
- [x] Decrease video speed (`CMD` `-`)

**Comment by giuscri** [comment]
`CMD` or `Ctrl`? Should that depend of the platform?, `darwin` or fallback?

[Apparently](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent#Properties) the event fired when a key is down has nice properties, such as `e.ctrlKey` and `e.altKey` -- not sure how to catch the `CMD` key.

This, combined with a change in the `playbackRate` property of `<video>` should do the last two tricks.

Pretty the same for the first two. This time, via 
```
dispatch('playbackJump', state.video.currentTime + /* or -*/ 10)
```

**Comment by ngoldman** [comment]
@giuscri in electron you can do `CmdOrCtrl` (which we're doing all over the place for current shortcuts). I would assume command is equivalent to control on non-mac platforms for discussion purposes.

**Comment by feross** [comment]
@giuscri Any interest in sending a PR to fix this issue?

**Comment by feross** [comment]
Fixed by @Lurk in #275!
